### PR TITLE
[FEATURE] Expose Config and query functions

### PR DESCRIPTION
**Pull request by jzjiang**
This PR exports the Config class and exposes the query functions through `client.Queries`. Also fixes up some function naming typos.

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -29,8 +42,27 @@ export default class Client {
...
+      {imageConnectionQuery},
+      {imageQuery},
+      {lineItemConnectionQuery},
+      {mailingAddressQuery},
+      {optionQuery},
+      {orderQuery},
+      {selectedOptionQuery},
+      {shippingRateQuery},
+      {variantConnectionQuery},
+      {variantQuery});
```
Comment (swalkinshaw) [inline src/client.js]: Nitpick: put closing `)` on its own line

File: src/client.js [hunk 2]
```diff
@@ -29,8 +42,28 @@ export default class Client {
...
+      {imageConnectionQuery},
+      {imageQuery},
+      {lineItemConnectionQuery},
+      {mailingAddressQuery},
+      {optionQuery},
+      {orderQuery},
+      {selectedOptionQuery},
+      {shippingRateQuery},
+      {variantConnectionQuery},
+      {variantQuery}
```
Comment (minasmart) [inline src/client.js]: Why can't this just be one hash?

Comment (jzjiang) [inline src/client.js]: whoops it should just be one hash

**Comment by jzjiang** [comment]
Updated with changes.

**Pull request merged**